src/harness/reference_models/dpa/dpa_mgr.py

```diff
@@ -254,6 +259,13 @@ def ComputeMoveLists(self):
     logging.debug('  protected points: %s', self.protected_points)
     pool = mpool.Pool()
     self.ResetLists()
+    # Detect the inside "inside grants", which will allow to
+    # add them into move list for sure later on.
+    inside_grants = set()
+    if self.geometry and not isinstance(self.geometry, sgeo.Point):
+      inside_grants = set(g for g in self._grants
+                          if sgeo.Point(g.longitude, g.latitude).within(self.geometry))
```

Comment:
it seems that this code will not handle the case when the CBSD is on one edge of the polygon.

object.within(other)¶
Returns True if the object’s boundary and interior intersect only with the interior of the other.

Reply:
Since the rest of the code distribute protection points along the edge, then you are right: we should consider the edge as being inside.
I have updated the code slightly to test for both inside and on the boundary, by using intersects() instead of within().